Vindicta.Altis/Unit/Unit.sqf

```diff
@@ -1767,7 +1767,19 @@ CLASS(UNIT_CLASS_NAME, "Storable")
 		} forEach _crewData;
 	} ENDMETHOD;
 
-	
+	//                             I S   E M P T Y
+	/*
+	Method: isEmpty
+	Returns true if there are no units in this vehicle or it is not a vehicle
+
+	Returns: bool
+	*/
+	METHOD("isEmpty") {
```

Comment:
Why is it needed? I can't see you use it anywhere in this PR.

Reply:
Also I am not sure if it will return true on player or false.

Reply:
It should be used for the bombers checking vehicles for targets. i.e. If a vehicle is enemy garrison but empty then its a good target for booby trap. Can't check the vic arma object directly because they aren't assigned to a side except in the Garrison itself.

Reply:
Yeah I guess makes sense!